File: app/src/main/java/ru/opalevapps/EveryGlic/AboutActivity.java
package ru.opalevapps.EveryGlic;

import android.content.pm.PackageInfo;
import android.content.pm.PackageManager;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

public class AboutActivity extends AppCompatActivity {
    String version;
    TextView tvAppVersion;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_info_about);

        tvAppVersion = findViewById(R.id.tvAppVersion);

        try {
            PackageInfo pInfo = this.getPackageManager().getPackageInfo(getPackageName(), 0);
            version = pInfo.versionName;
        } catch (PackageManager.NameNotFoundException e) {
            e.printStackTrace();
        }

        version = getString(R.string.version) + " - " + version;
        tvAppVersion.setText(version);
//        tvAppVersion.setText(String.format("%d - %s", R.string.version, version));

    }
}

File: app/src/main/java/ru/opalevapps/EveryGlic/ui/AboutActivity.kt
package ru.opalevapps.EveryGlic.ui

import android.content.pm.PackageManager
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.widget.TextView
import ru.opalevapps.EveryGlic.R

class AboutActivity : AppCompatActivity() {
    lateinit var version: String
    private lateinit var tvAppVersion: TextView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_info_about)

        tvAppVersion = findViewById(R.id.tvAppVersion)

        try {
            val packageInfo = this.packageManager.getPackageInfo(packageName, 0)
            version = packageInfo.versionName
        } catch (e: PackageManager.NameNotFoundException) {
            e.printStackTrace()
        }
        version = "${getString(R.string.version)} - $version"
        tvAppVersion.text = version
        //        tvAppVersion.setText(String.format("%d - %s", R.string.version, version));
    }
}